xml/System/Math.xml

```diff
@@ -616,6 +616,8 @@
 ## Remarks  
  Multiply the return value by 180/<xref:System.Math.PI?displayProperty=nameWithType> to convert from radians to degrees.  
   
+ This method calls into the underlying C runtime and the exact result or valid input range may differ between different Operating Systems or Architectures.  
```

Comment:
```suggestion
 This method calls into the underlying C runtime, and the exact result or valid input range may differ between different operating systems or architectures.  
```

Reply:
Applied to all occurrences of the sentence.